valora-inc/wallet
# chore(analytics): Include multichain assets for account_funded/liquidated event

### Description

Include multichain assets when deciding when to trigger account funded/liquidated analytics events.

### Test plan

Updated / added unit tests

Manual testing:

Be on the account funded/liquidated build on main
Send ETH to that wallet address
Check to make sure account funded event happens
Send CELO to that wallet address
Switch over to totalBalanceUsd build on main
Refresh transaction feed to trigger fetch_balances event
Check that totalCeloMainnetBalanceUsd and totalEthereumMainnetBalanceUsd are both > 0 on those events
Switch back to account funded/liquidated build.
Liquidate all CELO
Check that account liquidated event didn't happen (in the old case it would have because ETH not tracked, but here it should not since still have ETH)
Liquidate all ETH
Check that account liquidated event did happen
To check that events happened and super properties were included I was able to check the logs and also created a mixpanel dashboard to confirm.

This testing confirms that the super property shows up, and also checks that an account can be funded with ETH and the right event will be triggered, and liquidating all CELO assets does not trigger liquidated event, but liquidating ALL assets does trigger.

### Related issues

- Fixes #[ACT-1010]

### Backwards compatibility

Yes
